acme/acme/client.py

```diff
@@ -255,16 +258,23 @@ def retry_after(cls, response, default):
         :rtype: `datetime.datetime`
 
         """
-        retry_after = response.headers.get('Retry-After', str(default))
+        retry_after = response.headers.get('Retry-After', str(default)).strip()
         try:
             seconds = int(retry_after)
         except ValueError:
             # pylint: disable=no-member
-            decoded = werkzeug.parse_date(retry_after)  # RFC1123
-            if decoded is None:
+            when = parsedate_tz(retry_after)
```

Comment:
I haven't read the RFCs closely to figure out their relationships with each other. But `werkzeug.parse_date` is a thin wrapper on `email.utils.parsedate_tz`.